abdiff/config.py

```diff
@@ -31,6 +33,18 @@ def webapp_port(self) -> int:
         port = self.WEBAPP_PORT or "5000"
         return int(port)
 
+    @property
+    def transmogrifier_max_workers(self) -> int:
+        """Maximum number of Transmogrifier containers to run in parallel."""
+        max_workers = self.TRANSMOGRIFIER_MAX_WORKERS or 6
```

Comment:
Defaults to 6 parallel workers, striking a balance of performance and middle-of-the-road required system resources.  I believe all of our machine have 10 CPU cores, so theoretically 8-10 is quicker.